src/Utilities/FlowAnalysis/Analysis/CopyAnalysis/CopyAnalysis.cs

```diff
@@ -24,11 +25,16 @@ private CopyAnalysis(CopyDataFlowOperationVisitor operationVisitor)
             ControlFlowGraph cfg,
             ISymbol owningSymbol,
             WellKnownTypeProvider wellKnownTypeProvider,
-            PointsToAnalysisResult pointsToAnalysisResultOpt = null,
-            bool pessimisticAnalysis = true)
+            InterproceduralAnalysisKind interproceduralAnalysisKind = InterproceduralAnalysisKind.None,
```

Comment:
@dotpaul The primary changes needed for each specific dataflow analysis and/or rule to enable interprocedural analysis are as follows:
1. Pass in the `InterproceduralAnalysisKind` down to the analysis context, with a suitable default value.
2. Implement the abstract method `AbstractDataFlowAnalysisContext.ForkForInterproceduralAnalysis` on the analysis specific context type (similar to the change in the file `CopyAnalysisContext.cs` after this one).